app/models/claim.rb

```diff
@@ -306,10 +306,6 @@ def has_ecp_policy?
 
   private
 
-  def is_ecp_policy?
```

Comment:
Curious why this is still here? Thought we renamed before.

Reply:
Yes picked this up and have removed. Was a c/p error. Apologies